## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 437681`
Commit: `0152cdddb6fb313b66ead3675036807234bfa763`

---
### `x86_64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>nodePackages_latest.joplin</li>
    <li>nodePackages_latest.keyoxide</li>
    <li>nodePackages_latest.vega-cli</li>
    <li>nodePackages_latest.vega-lite</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 148 packages built:</summary>
  <ul>
    <li>corepack_24 (corepack_latest)</li>
    <li>github-runner</li>
    <li>nodePackages_latest.alex</li>
    <li>nodePackages_latest.audiosprite</li>
    <li>nodePackages_latest.awesome-lint</li>
    <li>nodePackages_latest.aws-cdk</li>
    <li>nodePackages_latest.bower</li>
    <li>nodePackages_latest.browser-sync</li>
    <li>nodePackages_latest.browserify</li>
    <li>nodePackages_latest.cdk8s-cli</li>
    <li>nodePackages_latest.cdktf-cli</li>
    <li>nodePackages_latest.clipboard-cli</li>
    <li>nodePackages_latest.coc-cmake</li>
    <li>nodePackages_latest.coc-docker</li>
    <li>nodePackages_latest.coc-emmet</li>
    <li>nodePackages_latest.coc-eslint</li>
    <li>nodePackages_latest.coc-explorer</li>
    <li>nodePackages_latest.coc-flutter</li>
    <li>nodePackages_latest.coc-git</li>
    <li>nodePackages_latest.coc-go</li>
    <li>nodePackages_latest.coc-haxe</li>
    <li>nodePackages_latest.coc-highlight</li>
    <li>nodePackages_latest.coc-html</li>
    <li>nodePackages_latest.coc-java</li>
    <li>nodePackages_latest.coc-jest</li>
    <li>nodePackages_latest.coc-json</li>
    <li>nodePackages_latest.coc-lists</li>
    <li>nodePackages_latest.coc-ltex</li>
    <li>nodePackages_latest.coc-markdownlint</li>
    <li>nodePackages_latest.coc-pairs</li>
    <li>nodePackages_latest.coc-prettier</li>
    <li>nodePackages_latest.coc-r-lsp</li>
    <li>nodePackages_latest.coc-rls</li>
    <li>nodePackages_latest.coc-rust-analyzer</li>
    <li>nodePackages_latest.coc-sh</li>
    <li>nodePackages_latest.coc-smartf</li>
    <li>nodePackages_latest.coc-snippets</li>
    <li>nodePackages_latest.coc-solargraph</li>
    <li>nodePackages_latest.coc-spell-checker</li>
    <li>nodePackages_latest.coc-sqlfluff</li>
    <li>nodePackages_latest.coc-stylelint</li>
    <li>nodePackages_latest.coc-sumneko-lua</li>
    <li>nodePackages_latest.coc-tabnine</li>
    <li>nodePackages_latest.coc-texlab</li>
    <li>nodePackages_latest.coc-tsserver</li>
    <li>nodePackages_latest.coc-ultisnips</li>
    <li>nodePackages_latest.coc-vetur</li>
    <li>nodePackages_latest.coc-vimlsp</li>
    <li>nodePackages_latest.coc-vimtex</li>
    <li>nodePackages_latest.coc-wxml</li>
    <li>nodePackages_latest.coc-yaml</li>
    <li>nodePackages_latest.coc-yank</li>
    <li>nodePackages_latest.code-theme-converter</li>
    <li>nodePackages_latest.conventional-changelog-cli</li>
    <li>nodePackages_latest.cpy-cli</li>
    <li>nodePackages_latest.create-cycle-app</li>
    <li>nodePackages_latest.cspell</li>
    <li>nodePackages_latest.csslint</li>
    <li>nodePackages_latest.dhcp</li>
    <li>nodePackages_latest.diff2html-cli</li>
    <li>nodePackages_latest.dotenv-vault</li>
    <li>nodePackages_latest.elasticdump</li>
    <li>nodePackages_latest.elm-oracle</li>
    <li>nodePackages_latest.emoj</li>
    <li>nodePackages_latest.emojione</li>
    <li>nodePackages_latest.esy</li>
    <li>nodePackages_latest.fast-cli</li>
    <li>nodePackages_latest.fauna-shell</li>
    <li>nodePackages_latest.fkill-cli</li>
    <li>nodePackages_latest.fleek-cli</li>
    <li>nodePackages_latest.forever</li>
    <li>nodePackages_latest.fx</li>
    <li>nodePackages_latest.gatsby-cli</li>
    <li>nodePackages_latest.grunt-cli</li>
    <li>nodePackages_latest.gulp</li>
    <li>nodePackages_latest.gulp-cli</li>
    <li>nodePackages_latest.he</li>
    <li>nodePackages_latest.hs-airdrop</li>
    <li>nodePackages_latest.imapnotify</li>
    <li>nodePackages_latest.js-beautify</li>
    <li>nodePackages_latest.js-yaml</li>
    <li>nodePackages_latest.jsdoc</li>
    <li>nodePackages_latest.jshint</li>
    <li>nodePackages_latest.json</li>
    <li>nodePackages_latest.json-diff</li>
    <li>nodePackages_latest.json-refs</li>
    <li>nodePackages_latest.json-server</li>
    <li>nodePackages_latest.jsonlint</li>
    <li>nodePackages_latest.jsonplaceholder</li>
    <li>nodePackages_latest.katex</li>
    <li>nodePackages_latest.lcov-result-merger</li>
    <li>nodePackages_latest.live-server</li>
    <li>nodePackages_latest.livedown</li>
    <li>nodePackages_latest.localtunnel</li>
    <li>nodePackages_latest.lua-fmt</li>
    <li>nodePackages_latest.madoko</li>
    <li>nodePackages_latest.makam</li>
    <li>nodePackages_latest.mastodon-bot</li>
    <li>nodePackages_latest.mathjax</li>
    <li>nodePackages_latest.meat</li>
    <li>nodePackages_latest.mocha</li>
    <li>nodePackages_latest.multi-file-swagger</li>
    <li>nodePackages_latest.nijs</li>
    <li>nodePackages_latest.node-gyp-build</li>
    <li>nodePackages_latest.node2nix</li>
    <li>nodejs_24 (nodePackages_latest.nodejs, nodejs_latest)</li>
    <li>nodejs_24.dev (nodePackages_latest.nodejs.dev, nodejs_latest.dev)</li>
    <li>nodejs_24.libv8 (nodePackages_latest.nodejs.libv8, nodejs_latest.libv8)</li>
    <li>nodePackages_latest.np</li>
    <li>nodePackages_latest.npm-merge-driver</li>
    <li>nodePackages_latest.nrm</li>
    <li>nodePackages_latest.parsoid</li>
    <li>nodePackages_latest.peerflix</li>
    <li>nodePackages_latest.peerflix-server</li>
    <li>nodePackages_latest.poor-mans-t-sql-formatter-cli</li>
    <li>nodePackages_latest.postcss</li>
    <li>nodePackages_latest.prebuild-install</li>
    <li>nodePackages_latest.pscid</li>
    <li>nodePackages_latest.pulp</li>
    <li>nodePackages_latest.purescript-language-server</li>
    <li>nodePackages_latest.purescript-psa</li>
    <li>nodePackages_latest.purs-tidy</li>
    <li>nodePackages_latest.purty</li>
    <li>nodePackages_latest.rimraf</li>
    <li>nodePackages_latest.rush</li>
    <li>nodePackages_latest.sass</li>
    <li>nodePackages_latest.semver</li>
    <li>nodePackages_latest.serve</li>
    <li>nodePackages_latest.sloc</li>
    <li>nodePackages_latest.smartdc</li>
    <li>nodePackages_latest.speed-test</li>
    <li>nodePackages_latest.svelte-check</li>
    <li>nodePackages_latest.tern</li>
    <li>nodePackages_latest.tiddlywiki</li>
    <li>nodePackages_latest.ts-node</li>
    <li>nodePackages_latest.tsun</li>
    <li>nodePackages_latest.ttf2eot</li>
    <li>nodePackages_latest.unified-language-server</li>
    <li>nodePackages_latest.uppy-companion</li>
    <li>nodePackages_latest.vercel</li>
    <li>nodePackages_latest.wavedrom-cli</li>
    <li>nodePackages_latest.webpack</li>
    <li>nodePackages_latest.wring</li>
    <li>nodejs-slim_24 (nodejs-slim_latest)</li>
    <li>nodejs-slim_24.dev (nodejs-slim_latest.dev)</li>
    <li>nodejs-slim_24.libv8 (nodejs-slim_latest.libv8)</li>
    <li>starkiller</li>
    <li>zipline</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>mealie</summary>
<pre>SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:71: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:92: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:100: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:121: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:129: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:155: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:182: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:208: requires ldap service in github actions
SKIPPED [1] tests/integration_tests/user_tests/test_user_login.py:234: requires ldap service in github actions
SKIPPED [6] tests/unit_tests/test_recipe_parser.py:18: Long Running API Test - manually run when updating the parser
FAILED tests/integration_tests/test_spa.py::test_spa_serve_recipe_with_meta[True] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_serve_recipe_with_meta[False] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_serve_recipe_with_meta_invalid_data[True] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_serve_recipe_with_meta_invalid_data[False] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_service_shared_recipe_with_meta[True-True] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_service_shared_recipe_with_meta[True-False] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_service_shared_recipe_with_meta[False-True] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_service_shared_recipe_with_meta[False-False] - Failed: async def functions are not natively supported.
FAILED tests/integration_tests/test_spa.py::test_spa_service_shared_recipe_with_meta_invalid_data - Failed: async def functions are not natively supported.
9 failed, 1170 passed, 18 skipped, 145 warnings in 150.20s (0:02:30)</pre>
</details>
<details>
<summary>nodePackages_latest.joplin</summary>
<pre>npm error gyp ERR! System Linux 6.12.41
npm error gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--update-binary" "--module=/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas/build/Release/canvas.node" "--module_name=canvas" "--module_path=/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas/build/Release" "--napi_version=10" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v137"
npm error gyp ERR! cwd /nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas
npm error gyp ERR! node -v v24.7.0
npm error gyp ERR! node-gyp -v v11.2.0
npm error gyp ERR! not ok 
npm error node-pre-gyp ERR! build error 
npm error node-pre-gyp ERR! stack Error: Failed to execute '/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node /nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --update-binary --module=/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas/build/Release --napi_version=10 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v137' (1)
npm error node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
npm error node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:508:28)
npm error node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1101:16)
npm error node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
npm error node-pre-gyp ERR! System Linux 6.12.41
npm error node-pre-gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--update-binary"
npm error node-pre-gyp ERR! cwd /nix/store/fqn2xhlx9s62131mn0j7j40rd0v2f50a-joplin-3.2.3/lib/node_modules/joplin/node_modules/canvas
npm error node-pre-gyp ERR! node -v v24.7.0
npm error node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm error node-pre-gyp ERR! not ok
|npm error A complete log of this run can be found in: /build/.npm/_logs/2025-08-29T09_01_08_949Z-debug-0.log
|</pre>
</details>
<details>
<summary>nodePackages_latest.keyoxide</summary>
<pre>npm error gyp ERR! System Linux 6.12.41
npm error gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--update-binary" "--module=/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas/build/Release/canvas.node" "--module_name=canvas" "--module_path=/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas/build/Release" "--napi_version=10" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v137"
npm error gyp ERR! cwd /nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas
npm error gyp ERR! node -v v24.7.0
npm error gyp ERR! node-gyp -v v11.2.0
npm error gyp ERR! not ok 
npm error node-pre-gyp ERR! build error 
npm error node-pre-gyp ERR! stack Error: Failed to execute '/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node /nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --update-binary --module=/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas/build/Release --napi_version=10 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v137' (1)
npm error node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
npm error node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:508:28)
npm error node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1101:16)
npm error node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
npm error node-pre-gyp ERR! System Linux 6.12.41
npm error node-pre-gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--update-binary"
npm error node-pre-gyp ERR! cwd /nix/store/s5npgai2w1wjszi8z7gka7c2xpyv1i9b-keyoxide-0.4.4/lib/node_modules/keyoxide/node_modules/canvas
npm error node-pre-gyp ERR! node -v v24.7.0
npm error node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm error node-pre-gyp ERR! not ok
\npm error A complete log of this run can be found in: /build/.npm/_logs/2025-08-29T09_00_51_996Z-debug-0.log
\</pre>
</details>
<details>
<summary>nodePackages_latest.vega-cli</summary>
<pre>npm error gyp ERR! System Linux 6.12.41
npm error gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--update-binary" "--module=/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas/build/Release/canvas.node" "--module_name=canvas" "--module_path=/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas/build/Release" "--napi_version=10" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v137"
npm error gyp ERR! cwd /nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas
npm error gyp ERR! node -v v24.7.0
npm error gyp ERR! node-gyp -v v11.2.0
npm error gyp ERR! not ok 
npm error node-pre-gyp ERR! build error 
npm error node-pre-gyp ERR! stack Error: Failed to execute '/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node /nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --update-binary --module=/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas/build/Release --napi_version=10 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v137' (1)
npm error node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
npm error node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:508:28)
npm error node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1101:16)
npm error node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
npm error node-pre-gyp ERR! System Linux 6.12.41
npm error node-pre-gyp ERR! command "/nix/store/49xq83bxi4gfzs02n0rlyynwy2r18b10-nodejs-24.7.0/bin/node" "/nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--update-binary"
npm error node-pre-gyp ERR! cwd /nix/store/7hd1z4km4w1lzb5k7q3rnjdblmvifxjp-vega-cli-5.32.0/lib/node_modules/vega-cli/node_modules/canvas
npm error node-pre-gyp ERR! node -v v24.7.0
npm error node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm error node-pre-gyp ERR! not ok
\npm error A complete log of this run can be found in: /build/.npm/_logs/2025-08-29T09_00_31_872Z-debug-0.log
\</pre>
</details>
</details>

--- a/pkgs/development/web/nodejs/v24.nix
+++ b/pkgs/development/web/nodejs/v24.nix
@@ -17,8 +17,8 @@ let
 in
 buildNodejs {
   inherit enableNpm;
-  version = "24.6.0";
-  sha256 = "8ad5c387b5d55d8f3b783b0f1b21bae03a3b3b10ac89a25d266cffa7b795e842";
+  version = "24.7.0";
+  sha256 = "cf74a77753b629ffebd2e38fb153a21001b2b7a3c365c0ec7332b120b98c7251";
   patches =
     (
       if (stdenv.hostPlatform.emulatorAvailable buildPackages) then
@@ -47,10 +47,18 @@ buildNodejs {
     ++ [
       ./configure-armv6-vfpv2.patch
       ./disable-darwin-v8-system-instrumentation-node19.patch
-      ./bypass-darwin-xcrun-node16.patch
       ./node-npm-build-npm-package-logic.patch
       ./use-correct-env-in-tests.patch
       ./bin-sh-node-run-v22.patch
+
+      # TODO: newer GYP versions have been patched to be more compatible with Nix sandbox. We need
+      # to adapt our patch to this newer version, see https://github.com/NixOS/nixpkgs/pull/434742.
+      (fetchpatch2 {
+        url = "https://github.com/nodejs/node/commit/886e4b3b534a9f3ad2facbc99097419e06615900.patch?full_index=1";
+        hash = "sha256-HFTabl92NPkBwXD0mUGDN+Gzabyi+Ph0kL0FEHHknbk=";
+        revert = true;
+      })
+      ./bypass-darwin-xcrun-node16.patch
     ]
     ++ lib.optionals (!stdenv.buildPlatform.isDarwin) [
       # test-icu-env is failing without the reverts

